src/components/hooks.js

```diff
@@ -4,6 +4,32 @@ import {
 import debounce from 'lodash/debounce';
 import { useGridResize, useEmitGridResize } from '../app/state/hooks';
 
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height,
+  };
+}
+
+export function useWindowDimensions() {
```

Comment:
Can you add a JSDoc comment to document the type of variable that this hook returns.